magento/adobe-stock-integration
# Log out from adobe stock &Display user profile information

### Description (*)
Sign-in link in the top right corner of the grid is changed to the Adobe username.
A popup with user profile information is shown when admin user clicks on the username
![deepin-screen-recorder_Select area_20190910114614](https://user-images.githubusercontent.com/51681547/64598601-b1b32500-d3c0-11e9-9c2e-a56b5b61c93d.gif)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#353: Log out from adobe stock
2. magento/adobe-stock-integration#355: Display user profile information when admin is logged in to adobe stock

### Manual testing scenarios (*)
1. Configure AdobeStockSdk
2. Login With user creadentials depend on -> https://github.com/magento/adobe-stock-integration/pull/357
3. See user information
4. Click Sign out
